### DISCUSSION TITLE
Allow ?ver=3039402 for shadow-root customCss

### DISCUSSION THREAD

**Discussion by paulschek** [add2cal/add-to-calendar-button#548] [Ideas / Requests]
Howdy,

Trying to rotate version of the CSS file in case of hard case browsers like Safari that never flush cache. 

Would be nice if I could insert a CSS with a version number and therefore rotate the CSS in browser's cache. 

```
$stylesheetDirectory = esc_url( get_stylesheet_directory_uri()) . "/assets/css/atcb.css" . "?ver=" . substr(str_shuffle('0123456789'), 0, 5);

$shortcode = "[add-to-calendar-button customCss=\"$stylesheetDirectory\" --btn-background-image=\"none\" --btn-padding-y=\"0\" --btn-padding-x=\"0\" styleLight=\"--btn-hover-background=\"none\";--btn-background: #fff; --btn-text: #012841; --font: 'DM Sans', sans-serif;\" buttonStyle=\"custom\" hideIconButton hideBackground listStyle=\"overlay\" availability=\"busy\" hideCheckmark label=\"$label\" trigger=\"click\" hideBranding=\"true\"  name=\"$name\" options=\"$options\" startDate=\"$startDate\" endDate=\"$endDate\" startTime=\"$startTime\" endTime=\"$endTime\" timeZone=\"$timeZone\"  description=\"$description\"]";
```

^^^ here is an example.

As soon as the ver part is added, the shadow root stops being generated. 

I tried both @ and ? - both break the shadow-root elements.

**Comment by paulschek** [comment]
After enabling debug mode, I see this error in the console:

![image](https://github.com/add2cal/add-to-calendar-button/assets/145492247/53168dce-8c62-475f-8f53-02333e7bc1e2)

**Comment by jekuer** [comment]
Like it! Will do.

**Comment by jekuer** [comment]
Now possible with v2.5.10.
core script for now - React and WordPress adjustments will follow asap.
